test/SlowTests/Issues/RavenDB_8355.cs

```diff
@@ -0,0 +1,392 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using FastTests;
+using Orders;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Sorters;
+using Raven.Client.Documents.Queries.Sorting;
+using Raven.Client.Exceptions;
+using Raven.Client.Exceptions.Documents.Sorters;
+using Raven.Client.Http;
+using Raven.Server.Documents.Queries;
+using Sparrow.Json;
+using Xunit;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_8355 : RavenTestBase
+    {
+        private const string SorterCode = @"
+using System;
```

Comment:
Can probably put these in files and then use them as embedded resources.
Would probably be more readable

Reply:
Done